### DISCUSSION TITLE
Est-ce que FRW fonctionne ?

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#255] [Annonces]
Est-ce que FRW fonctionne ?

**Comment by Hutson48** [comment]
Bonjour,
Vous pouvez suivre le statut de FRW dans la section "README" de la page d'accueil de notre GitHub: https://github.com/MTESSDev/FRW.
Statut UP signifie que la solution fonctionne.  Cela dit, il y a effectivement eu une interruption tout à l'heure.

![image](https://github.com/MTESSDev/FRW/assets/158497985/095caf58-aa71-4621-8832-6c783751cb8e)
